lib/kuality-kfs/data_objects/financial_processing/disbursement_voucher_object.rb

```diff
@@ -15,8 +15,7 @@ class DisbursementVoucherObject < KFSDataObject
 
   def defaults
     super
-    super.merge!(get_aft_parameter_values_as_hash(ParameterConstants::DEFAULTS_FOR_DISBURSEMENT_VOUCHER))
-         .merge!(default_accounting_lines)
+    super.merge!(default_accounting_lines)
```

Comment:
Based on the comments associated with one of the other "super" calls, I believe those explicit invocations of the parent class default method was needed to ensure the data values initialized by the parent class were in fact properly set when the code was split between a base code project and our specific implementation of the AFTs.  Part of the current upgrade work was to combine those two projects into one. I suspect removing that extra (and now unnecessary "super" call because of the subsequent merge call) was missed during that previous effort. I have found three occurrences and have removed them.  Please re-review.